src/harmony/ActionWrapper/ActionWrapper.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { ReactNode } from 'react';
+
+import s from './ActionWrapper.module.css';
+
+interface ActionWrapperProps {
+    action?: ReactNode;
+    children?: ReactNode;
+}
+
+export const ActionWrapper = ({ action, children }: ActionWrapperProps) => {
```

Comment:
why not use the `iconRight` prop of `Badge` component? 

https://github.com/taskany-inc/bricks/blob/main/src/harmony/Badge/Badge.tsx#L29

Reply:
because the `action` needs to be shown only on the hover, and `User` component cannot be used with a `Badge`

Reply:
Pass additional element with passed action node, toggle visibility on additional element